src/index.html

```diff
@@ -21,7 +24,412 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="/"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
```

Comment:
The 'is-active' class suggests that this is the current active page. If this is a single page application, consider implementing a way to dynamically assign this class to the appropriate navigation link. If it's a multi-page application, ensure that the class is only used on the current page's navigation link.